### PR TITLE
Feature design small changes

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* [Isa comments (small changes)](https://www.notion.so/xmartlabs/Isa-comments-small-changes-2e07ccb7e275439b8580f7e53a85e51e?pvs=4)

## Type of change

* [ ] Fix
* [X] Story
* [ ] Chore

## Description of the change

I have changed the original styles to match the new ones. The blog categories, blog cover photos and some minor changes.

## Screenshot/Execution

https://user-images.githubusercontent.com/107500715/231574512-7a3987b6-b3eb-400c-9de0-3fa3e5ac604b.mp4

https://user-images.githubusercontent.com/107500715/231574517-76719900-d507-4cb7-81cc-58f94c2c02fe.mp4

Captura desde 2023-04-12 17-14-37

## Related PRs
Links to the related PRs

### CODE REVIEW

File: src/components/more-blogs-section/more-blogs-section.module.scss [hunk 1]
```diff
@@ -28,6 +28,12 @@ span {
 .cardStyles {
   width: 25%;
   margin-bottom: 5rem;
+
+  p {
+    position: absolute;
+    top: 13.5rem;
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply src/components/more-blogs-section/more-blogs-section.module.scss:34]: Done

[the remark this answers is not in the document]
Comment (lauta2912) [reply src/components/more-blogs-section/more-blogs-section.module.scss:34]: This still has no parent with `position: relative`, doesn't it?

File: src/components/tags/tags.module.scss [hunk 2]
```diff
@@ -1,22 +1,32 @@
 @import "/src/css/variables";
 
+.tagsContainer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+// .tag {
```
Comment (lauta2912) [inline src/components/tags/tags.module.scss:9]: let's remove this commented code

File: src/components/tags/tags.module.scss [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
 .tag {
   border: 1.5px solid rgba(155, 155, 155, 0.5);
   border-radius: 6.49875px;
-  height: 1.6rem;
   margin: 0 1rem 1rem 0;
-  text-align: center;
-  padding: 0 1rem;
-  line-height: 1.7rem;
+  text-align: left;
+  padding: 0.3rem 0.3rem;
```
Comment (lauta2912) [inline src/components/tags/tags.module.scss:25]: When you set both vertical and horizon axes with the same value, you can simplify it with the value only once and the browser is gonna use it for all axes
```suggestion
  padding: 0.3rem;
```

File: src/components/tags/tags.module.scss (hunk 2 above)
Comment (Melissa-Martinez-XL) [reply src/components/tags/tags.module.scss:9]: Done

File: src/components/tags/tags.module.scss (hunk 3 above)
Comment (Melissa-Martinez-XL) [reply src/components/tags/tags.module.scss:25]: Done

**Review by lauta2912 — APPROVED** [review]

**Pull request merged**